pymilvus/orm/iterator.py

```diff
@@ -257,6 +247,20 @@ def merge(self, others: List[Hits]):
             for other in others:
                 self._results.append(other)
 
+    def ids(self):
+        ids = []
+        for res in self._results:
+            for hit in res:
+                ids.append(hit.id)
+        return ids
```

Comment:
Maybe ids can be cached if it's immutable, and then it will be faster next time `ids` is called.

Reply:
True, it can be faster, but SearchPage is not immutable as it supports merging other SearchPages. Once changed, cached ids and distances need to be calculated once more. And currently, ids and distances methods are only called by e2e test. I suppose most users would only use get__item method to get the whole entity so caching these two list inside SearchPage would consume double-size memory and may not be very cost-effective